### DISCUSSION TITLE
Release cadence?

### DISCUSSION THREAD

**Discussion by jakehilton** [Grokzen/redis-py-cluster#443] [General]
Wondering if there is going to be a predictable release schedule for this library provided there are changes sitting on master.

Which brings me to my next question.. There is a fix sitting on the master branch that is biting us in our use of the lib. Curious when the next version will be cut? I'd much rather pull from a public endpoint than patch it myself. :)

Thanks for the great library!

**Comment by Grokzen** [comment]
@jakehilton No there is no set release cadence and i will not submit to a specific release schedule or similar. I do know that releases is far between these days but they get out eventually. They are slowing down due to work/life balance mostly as it takes a lot to further develop this lib to advance the features etc. Right now most of the time will not be put into the 2.1.x release track but i want to focus much more on the 3.0.0 release as that will be a bigger rewrite of the internals of the lib to something new. But there is no timeline for that but you can follow along in the issues marked 3.0.0 and the 3.0.0 branch i got up.

**Comment by jakehilton** [comment]
I appreciate the response and can totally understand the work/life balance. I was mainly interested in the changes that had been on the master branch for a few months that were helpful to my use case. I do see you just recently cut a new release which is great! Thanks for all your help.

**Reply by Grokzen** [reply]
No problems :) yeah the changes that was in needed to be pushed out. The only parts that would be pushed into 2.1. now is the readonly fixes but all other stuff would go into 3.0 for now
